lstchain/reco/dl1_to_dl2.py

```diff
@@ -294,28 +317,48 @@ def build_models(filegammas, fileprotons,
 
     config = replace_config(standard_config, custom_config)
     events_filters = config["events_filters"]
-    regression_features = config["regression_features"]
 
     df_gamma = pd.read_hdf(filegammas, key=dl1_params_lstcam_key)
     df_proton = pd.read_hdf(fileprotons, key=dl1_params_lstcam_key)
 
     df_gamma = utils.filter_events(df_gamma, filters=events_filters)
     df_proton = utils.filter_events(df_proton, filters=events_filters)
 
+    #Set source-dependent paramters
+    if src_dependent:
+        print("source-dependent analysis is activated (source position = {}deg)".format(expected_src_pos))
+        
+        focal_length = 28 * u.m
+        expected_src_pos = focal_length.value * np.tan(np.deg2rad(expected_src_pos))
```

Comment:
I would explicitly change the name `expected_src_pos` to something like `expected_src_pos_m`, or better, define `expected_src_pos` in l.274 as `expected_src_pos_deg`

Reply:
for the moment I deleted it